zerver/lib/email_notifications.py

```diff
@@ -766,6 +835,17 @@ def enqueue_welcome_emails(user: UserProfile, realm_creation: bool = False) -> N
             delay=followup_email_delay(user, days=2),
         )
 
+    if send_zulip_guide_followup:
+        send_future_email(
+            "zerver/emails/followup_zulip_guide",
```

Comment:
Thinking about naming, maybe we should call these `welcome_zulip_guide`? We call this category of emails "welcome emails" in a few places, and that feels more appropriate than "followup" as a thing to add more references to in the code.

I think the other sensible option is to call these `onboarding_zulip_guide`, and plan to rename things towards that. (The only thing that's nontrivial to change is the `realm_enable_welcome_emails` API field / database model name, and we could just lazily not rename that). I think it'd be a name that more obviously conveys what these are, so maybe that's the best plan? 

In other words, I propose we make this PR use `onboarding_` rather than `followup_` for all the new stuff, and we maybe do a follow-up PR to sweep all the easy things we can rename towards that, but try to not spend a lot of time on it.

Reply:
I like `onboarding_` and that plan in general.

Reply:
Updated for `onboarding_zulip_guide` for the email name.